### PR TITLE
Provide a better message when no keys are available.

### PR DESCRIPTION
**Pull request by rlhawk**
As described in #34.

### CODE REVIEW

File: src/Controller/KeyListBuilder.php [hunk 1]
```diff
@@ -33,4 +33,14 @@ public function buildRow(EntityInterface $entity) {
     return $row + parent::buildRow($entity);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function render() {
+    $entities = $this->load();
+    $build = parent::render();
+    $build['table']['#empty'] = t('No keys are available. <a href="@link">Add a key</a>.', array('@link' => \Drupal::url('entity.key.add_form')));
```
Comment (mradcliffe) [inline src/Controller/KeyListBuilder.php]: Does this output a real link these days with all the SafeMarkup changes in Twig? That could be a good assertion for integration/functional test.

Comment (rlhawk) [inline src/Controller/KeyListBuilder.php]: I modeled it after a similar message in core, so it should be correct, unless there are some very recent changes to Twig that would invalidate it. I agree that a test would be useful.

**Comment by mradcliffe** [comment]
This looks good to me.

**Comment by nerdstein** [comment]
Looks good to me. Merging

**Pull request merged**